server/models/Review.js

```diff
@@ -1,23 +1,32 @@
 const mongoose = require("mongoose");
 
-const reviewSchema = new mongoose.Schema(
-  {
-    user: {
-      firstName: String,
-      lastName: String,
-      profileImg: String,
-    },
-    rating: {
-      type: Number,
-      min: 1,
-      max: 5,
-      required: true,
-    },
-    message: String,
+const { Schema } = mongoose;
+const { ObjectId } = Schema;
+
+const reviewSchema = new mongoose.Schema({
+  reviewerUserId: {
+    type: ObjectId,
+    ref: "user",
+    unique: true,
```

Comment:
This means a user can only write one review ever, if you're trying to prevent having multiple reviews of and by the same user you could either do that in the controller or have a separate unique value that combines the two ids

Reply:
Got it.